.gitignore

```diff
@@ -8,6 +9,7 @@ java/build/*
 .idea/vcs.xml
 .idea/misc.xml
 .idea/jsLibraryMappings.xml
+
 cs340.properties
```

Comment:
(I'm mostly curious why `cs340.properties` is mentioned in the `.gitignore` when I never see it anywhere.